resources/RELEASE_2023_02_27.md

```diff
@@ -0,0 +1,16 @@
+Day 1 patch!
+_ _
+⫸ Updated [Teryl](https://discord.com/api/oauth2/authorize?client_id=948377583626637343&permissions=1074004032&scope=applications.commands%20bot):
+　⪢ Added autocomplete for </reminders update:1078828281949859985> and </reminders delete:1078828281949859985>.
```

Comment:
Don't the IDs change when re-registering?

Reply:
No, as long as the names don't change, they're kept, that's why Nekokai's and Iriss's command IDs are 18 digits long.